### DISCUSSION TITLE
Can you build a terminal multiplexer using bubble-tea?

### DISCUSSION THREAD

**Discussion by cakir-enes** [charmbracelet/bubbletea#678] [Q&A]
Hey, so Im aware this exists https://github.com/charmbracelet/bubbletea/issues/171#issuecomment-1097003263 which is nice, but since `ExecProcess`  pauses the program its not suitable to use (you wouldnt be able to swith to other panes while you are running your vim on one pane right?)

To give more context, I wont be trying to build a full blown terminal multiplexer more like this https://github.com/pvolok/mprocs but you can peek to other running commands output like showing the output in a modal while you are in your editor

Can I build sth like this using bubble-tea or going lower level  (idk tcell maybe?) is a must?

**Comment by different55** [comment]
I believe you _can_, but BubbleTea won't do the multiplexing for you. Same for tcell I believe. You can use either to make your UI, but the dirty work of managing the child processes is left to you. Don't quote me on this though.
